### PR TITLE
Node notes cli

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
https://user-images.githubusercontent.com/117682160/214764884-fb6928bc-4928-43a8-bc8a-08be02c52d44.mp4

https://user-images.githubusercontent.com/117682160/214764926-74c496fe-7e70-42a5-ab8e-bf38d3c02367.mp4

https://user-images.githubusercontent.com/117682160/214764972-6cd7c026-f900-4643-963c-b08b6e04062c.mp4

https://user-images.githubusercontent.com/117682160/214765074-0c48e143-02d3-457a-8a13-c727d42bb3be.mp4

### CODE REVIEW

**Comment by joseph-w-valdez** [comment]
https://user-images.githubusercontent.com/117682160/215362165-fcf1790f-9733-42a5-a205-396211aa7793.mp4


https://user-images.githubusercontent.com/117682160/215362166-b66eb49a-0662-47db-b236-3ce48d2002d5.mp4


https://user-images.githubusercontent.com/117682160/215362170-2c462692-517f-4827-acc7-30344aeed0c8.mp4


https://user-images.githubusercontent.com/117682160/215362172-f87ba10c-ac47-4e2e-b549-620d5e22dd2d.mp4

**Comment by joseph-w-valdez** [comment]
ready for review!

File: node-notes-cli/delete.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const write = require('./write');
+const remove = (data, index) => {
+  if (index > 0 && index % 1 === 0) {
```
Comment (RobertGardner) [inline node-notes-cli/delete.js:3]: `index % 1 === 0` is a clever way to make sure you have an integer, but it's a bit obscure. Sometimes clarity is better than cleverness. :-)

**Review by RobertGardner — APPROVED** [review]

File: node-notes-cli/delete.js (hunk 1 above)
Comment (joseph-w-valdez) [reply node-notes-cli/delete.js:3]: I'll bear this in mind. Ty!

**Pull request merged**